src/main/proto/DataTile.proto

```diff
@@ -102,7 +131,7 @@ message TileSummary{
     string dataset_name = 3;
     string granule = 4;
     string dataset_uuid = 5;
-    string data_var_name = 6;
+    repeated string data_var_name = 6;
```

Comment:
is changing from single value to repeated a backwards-compatible change?

Reply:
yup.. it doesn't. forgot about this one.. Let me see if I can work with single data_var_name instead of an array.